package.json

```
@@ -43,26 +43,30 @@
   },
   "homepage": "http://fastify.dev/",
   "devDependencies": {
-    "@fastify/secure-session": "^7.0.0",
-    "@types/node": "^20.1.0",
-    "fastify": "^4.0.0-rc.2",
-    "prettier": "^3.0.0",
-    "rimraf": "^5.0.0",
-    "tap": "^16.0.0",
-    "tap-mocha-reporter": "^5.0.3",
-    "ts-node": "^10.7.0",
+    "@fastify/secure-session": "^7.1.0",
+    "@types/node": "^20.11.28",
+    "fastify": "^4.26.2",
+    "prettier": "^3.2.5",
+    "rimraf": "^5.0.5",
+    "tap": "^18.7.1",
+    "tap-mocha-reporter": "^5.0.4",
+    "ts-node": "^10.9.2",
     "tslint": "^6.1.3",
     "tslint-config-prettier": "^1.18.0",
-    "typescript": "^5.0.2"
+    "typescript": "^5.4.2"
   },
   "dependencies": {
-    "fastify-plugin": "^4.0.0"
+    "fastify-plugin": "^4.5.1"
   },
   "files": [
     "/lib",
     "/typings"
   ],
   "publishConfig": {
     "access": "public"
+  },
+  "tap": {
+    "plugin": [],
+    "files": ["./tests/**/*.ts"]
```

Please move the config to `.taprc` for consistency

```suggestion
  }
```